kernel_gateway/services/sessions/handlers.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+import notebook.services.sessions.handlers as notebook_handlers
+from ...mixins import TokenAuthorizationMixin, CORSMixin
+
+default_handlers = []
+for path, cls in notebook_handlers.default_handlers:
+    # Everything should have CORS and token auth
+    bases = (TokenAuthorizationMixin, CORSMixin, cls)
```

Comment:
I think we've got `--KernelGatewayApp.list_kernels=False` as the param name and default value at the moment. Maybe we deprecate that, make it cover both kernels and sessions listing, and add `--KernelGatewayApp.discovery=False` as a better name to cover both?